python/lib/sift_py/rest.py

```diff
@@ -2,6 +2,8 @@
 
 from typing_extensions import NotRequired
 
+from sift_py.grpc.transport import _clean_uri
```

Comment:
non-blocking: we should move the _clean_uri into the top level utils module later